releasenotes/notes/25617.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: istioctl
+issue:
+- 25581
+
+# No releaseNotes
```

Comment:
If there is no notes we probably don't need a file here. Otherwise we are going to end up with 10k files with no purpose in this folder

Reply:
@howardjohn Style question.  When I had no notes there was a PR test failing.  It wasn't a mandatory test, but I decided to do this to make it pass.  Should I have instead labeled the PR as not needing a release note?

Reply:
@esnible If you look at the test results, the appropriate way to make the test pass if there are no release notes is to add the label.

That said, this looks like a change to istioctl, which is something the customer would see, so should have a release note associated with it.

Reply:
See https://istio.io/latest/news/releases/1.5.x/announcing-1.5/change-notes/#istioctl for examples.